setup.cfg

```diff
@@ -73,3 +73,6 @@ exclude_lines =
     raise ImportError
     assert
     pass
+
+[pytest]
+addopts = --env=MODIN_USE_PLAIN_PANDAS_QUERY_COMPILER=True
```

Comment:
Why is this needed?

Reply:
Removed this, it was to run pytests in VScode for local development.